test/tools/htmlreport/check.sh

```diff
@@ -26,21 +26,22 @@ if [ -z "$PYTHON" ]; then
     PYTHON=python
 fi
 
+PROJECT_ROOT_DIR=$(git rev-parse --show-toplevel)
```

Comment:
This won't work if you are running this from a source package without a repo.

In other scripts we use `DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"` to get the script dir. And then traverse back from that.

Reply:
Should be fixed now.